### ISSUE TITLE
Can't create /data folder on mac for RTP

### ISSUE THREAD

**Issue by kevincrans** [EasyRPG/Player#3076]
Thank you for supporting EasyRPG Player.
Before creating an issue, please verify that your bug still occurs in the "continuous builds" of the Player.
You can get them at the bottom of our download section: https://easyrpg.org/player/downloads

---
Please fill in the following fields before submitting an issue:
#### Name of the game:
Any
#### Player platform:
Mac
#### Describe the issue in detail and how to reproduce it:
On Macs since just before Big Sur, root users don't have root folder permissions.
I think EasyRPG should use a different folder for them, not cause Apple is right, but /data is indeed not a typical POSIX folder anybody uses, so using it, even with a symlink or jailbreaking/disabling SIP, is bad practice.
Also, Wine sucks on mac, so besides the register running, it needs to be extracted by hand.

**Comment by Ghabry** [comment]
@kevincrans it should be possible to use the Linux paths for the RTP even on macOS. They are ``/usr/share/rtp/`` or ``/usr/local/share/rtp`` (2000 and 2003). Can you test if this works?

**Comment by fdelapena** [comment]
On macOS, local apps should store data and config at `~/Library/Application Support`, local user will be able to read and write from there without "full disk access" permission, so storing RTP there looks a good place instead of providing a some .app bundle for the Applications directory.

**Comment by kevincrans** [comment]
> @kevincrans it should be possible to use the Linux paths for the RTP even on macOS. They are `/usr/share/rtp/` or `/usr/local/share/rtp` (2000 and 2003). Can you test if this works?

No, the /usr/local/share permissions can't be obtained by the root user (Administrator). Sudo doesn't give system permission.

So the only options are /opt and what @fdelapena says.

**Comment by kevincrans** [comment]
I edited my comment
